notebooks/runtime_hello_widget.py

```
@@ -0,0 +1,51 @@
+        0,
+        len(NAMES) - 1,
+        value=0,
+        label="name index",
+        show_value=True,
+    )
+    name_index
+    return (name_index,)
+
+
+@app.cell(hide_code=True)
+def _(name_index):
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `_`

(ANN202)